guibot/target.py

```diff
@@ -558,17 +558,20 @@ def save(self, steps_filename):
             elif step_backend in ["cascade", "deep"]:
                 data = data_and_config.data_file
             elif step_backend == "text":
+                # special case - dynamic text without a filename
+                # save only the matchfile and add the corresponding line
+                if not data_and_config.text_file:
+                    matchfile = str(data_and_config) + ".match"
+                    Target.save(data_and_config, matchfile)
+                    save_lines.append(data_and_config.value + "\t" + matchfile + "\n")
+                    continue
```

Comment:
Could this break in case there is text including tabs? I guess it could be rare type of text but will it work with all text? For instance text with new lines and various characters that might break the parsing?

Reply:
Good question. The `Chain` target [doesn't accept values with tabs](https://github.com/intra2net/guibot/blob/master/guibot/target.py#L520), so it will throw an exception if there's an extra tab. Also, new lines would be interpreted as another step, thus a malformed line in this case.
Added two more tests covering such cases :slightly_smiling_face: